frontend/app/routes/protected/renew/$id/$childId/demographic-survey.tsx

```diff
@@ -48,6 +48,12 @@ export async function loader({ context: { appContainer, session }, request, para
   const securityHandler = appContainer.get(TYPES.routes.security.SecurityHandler);
   await securityHandler.validateAuthSession({ request, session });
 
+  const { ENABLED_FEATURES } = appContainer.get(TYPES.configs.ClientConfig);
+  const demographicSurveyEnabled = ENABLED_FEATURES.includes('demographic-survey');
+  if (!demographicSurveyEnabled) {
+    throw new Response(null, { status: 404 });
+  }
```

Comment:
We should be able to use the `securityHandler.validateFeatureEnabled(..)` convenience function here instead.

(same comment for `frontend/app/routes/protected/renew/$id/demographic-survey.tsx`)